languages/rust/exercises/concept/vec-macro/.meta/example.rs

```diff
@@ -0,0 +1,20 @@
+/// vec-macro for creating empty vectors, and vectors of particular size, with vec!
+
+/// Create an empty vector
+fn create_empty() -> Vec<u8> {
+    vec![]
+}
+
+/// Create a buffer of `count` zeroes.
+///
+/// Applications use these for filling and modifying data
```

Comment:
```suggestion
/// Applications often use buffers when serializing data to send over the network.
```

More precise example.